app/vue/src/server/config/webpack.config.js

```diff
@@ -52,7 +52,7 @@ export default function() {
       // Based on this CRA feature: https://github.com/facebookincubator/create-react-app/issues/253
       modules: ['node_modules'].concat(nodePaths),
       alias: {
-        'vue$': 'vue/dist/vue.esm.js'
+        'vue': path.resolve(path.join(__dirname, '../../../node_modules', 'vue/dist/vue.esm.js'))
```

Comment:
@kazupon I fyou could try to run the example s storybook and see if all the stories are working :)